packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -275,6 +275,8 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 
 	const { defaultPaidPlan, defaultFreePlan, planPrices } = usePlans();
 
+	const [ nonDefaultPaidPlan, setNonDefaultPaidPlan ] = useState< Plan | null | undefined >( null );
```

Comment:
Small nitpick, but we could simplify the possible values of `nonDefaultPaidPlan` if we initialized it to `undefined`:

```suggestion
	const [ nonDefaultPaidPlan, setNonDefaultPaidPlan ] = useState< Plan | undefined >( undefined );
```